services/promises.js

```diff
@@ -19,6 +19,31 @@ function defer() {
 }
 
 /**
+ * bluebird.props with native promises
+ * @param {object} props
+ * @returns {Promise}
+ */
+function props(props) {
```

Comment:
This works the same as `bluebird.props`, but I didn't want to pull in the _whole_ bluebird lib just for one function.